dap_prinz_green_jobs/analysis/ojo_analysis/aggregate_by_region.py

```diff
@@ -0,0 +1,93 @@
+from dap_prinz_green_jobs.analysis.ojo_analysis.process_ojo_green_measures import *
+from dap_prinz_green_jobs.getters.data_getters import save_to_s3
+from dap_prinz_green_jobs import BUCKET_NAME
+
+from datetime import datetime
+
+if __name__ == "__main__":
+    job_id_col = "job_id"
+    production = "True"
+    config = "base"
+    skills_date_stamp = "20230914"
+    occ_date_stamp = "20231002"
+    ind_date_stamp = "20231013"
+
+    (
+        skill_measures_df,
+        occs_measures_df,
+        inds_measures_df,
+        soc_name_dict,
+    ) = load_ojo_green_measures(
+        production, config, skills_date_stamp, occ_date_stamp, ind_date_stamp
+    )
+
+    all_green_measures_df = merge_green_measures(
+        skill_measures_df, occs_measures_df, inds_measures_df, soc_name_dict
+    )
+
+    # Add some additional data
+
+    all_green_measures_df = add_salaries(all_green_measures_df, job_id_col=job_id_col)
+    all_green_measures_df = add_locations(all_green_measures_df, job_id_col=job_id_col)
+    all_green_measures_df = add_sic_info(all_green_measures_df)
+
+    all_skills_df = create_skill_df(skill_measures_df)
+
+    agg_itl_by = "itl_3_code"  # itl_2_code or itl_3_code
+
+    itl_aggregated_data = create_agg_data(
```

Comment:
something i'd randomly be careful with/check is that London isn't always merged appropriately (a bug in the locations algorithm?) - I can add an issue to the `ojd_daps_locations` repo to investigate and mention to jack / might just want to have a cleaner in a utils somewhere